File: aspnet-core/src/doan.ProjectManagement.Application.Contracts/SystemSettings/Dto/SystemSettingDto.cs
using doan.ProjectManagement.Enum;
using System;
using Volo.Abp.Application.Dtos;

namespace doan.ProjectManagement.SystemSettings.Dto
{
    public class SystemSettingDto : FullAuditedEntityDto<Guid>
    {
        public string Key { get; set; }
        public string Value { get; set; }
        public string Description { get; set; }
        public Status Status { get; set; }
        public DateTime? EffectDate { get; set; }
        public DateTime? ExpireDate { get; set; }
    }
}

File: applications/teacher-web/src/app/shared/services/system-configuration/system-setting/models/system-setting-dto.ts
import { Status } from './status';
import { EntityDto } from "@abp/ng.core";

export class SystemSettingDto extends EntityDto<string>{
  key: string;
  value?: String;
  description?: string;
  status?: Status;
  effectDate?: Date;
  expireDate?: Date;

  constructor(initialValues: Partial<SystemSettingDto>) {
    super(initialValues);
  }
}
